docs/UserGuide.md

```diff
@@ -62,26 +62,21 @@ Amigos is designed to help you keep track of the friends in your life.
 ### Adding a friend: `addfriend`
 
 Adds a new friend to Amigos. A friend has:
-* ***(compulsory)*** name
-* *(optional)* phone number
-* *(optional)* email
-* *(optional)* address
-* *(optional)* description
 
-**Format**: `addfriend n/[NAME]  p/[PHONE_NUMBER] e/[EMAIL]  a/[ADDRESS] d/[DESCRIPTION]`
+**Format**: `addfriend n/[NAME]  [p/PHONE_NUMBER] [e/EMAIL]  [a/ADDRESS] [d/DESCRIPTION] [t/TAG]…`
```

Comment:
uhh it still doesn't follow the formatting - it's supposed to look smth like this:

- edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​

See: https://se-education.org/addressbook-level3/UserGuide.html

Do remember to update the table accordingly once you're done.

Reply:
Apologies! Thank you for pointing it out.